swift/ql/test/extractor-tests/expressions/expressions.swift

```diff
@@ -181,3 +181,7 @@ let optChain = \Foo.opt?.opt
 let optChainWrap = \Foo.opt?.value
 let slf = \Int.self
 let tupleElement = \(Int, Int).0
+
+func makeTuple<each T>(_ t: repeat each T) -> (repeat each T) {
+  return (repeat each t)
+}
```

Comment:
They _really_ didn't want the C++ `...` syntax, so they just relabeled it, eh? :laughing: